pkg/kubelet/dockertools/manager.go

```diff
@@ -651,7 +656,7 @@ func (dm *DockerManager) runContainer(
 	// while the Kubelet is down and there is no information available to recover the pod. This includes
 	// termination information like the termination grace period and the pre stop hooks.
 	// TODO: keep these labels up to date if the pod changes
-	labels := newLabels(container, pod, restartCount)
+	labels := newLabels(container, pod, restartCount, dm.enableCustomMetrics)
```

Comment:
We need to add a proper label for cAdvisor for custom metrics. When adding new labels, we need to decide, if custom metrics support is on and if we want to add a special label for custom metrics or not.